scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
-  - tanzu-package-repo-global
-  - tanzu-system-ingress
-  - tap-gui
-  - tap-install
-  - tap-telemetry
-  - tekton-pipelines
-  - triggermesh
-  ```
-
-The following is an example `ClusterImagePolicy`:
-
-  ```yaml
-  ---
-  apiVersion: signing.apps.tanzu.vmware.com/v1beta1
-  kind: ClusterImagePolicy
-  metadata:
-      name: image-policy
-  spec:
-    verification:
-      exclude:
-        resources:
-          namespaces:
-          - kube-system
-          - <TAP system namespaces>
-      keys:
-      - name: first-key
-        publicKey: |
-          -----BEGIN PUBLIC KEY-----
-          ...
-          -----END PUBLIC KEY-----
-      images:
-      - namePattern: registry.example.org/myproject/*
-        keys:
-        - name: first-key
-      - namePattern: registry.example.org/authproject/*
-        secretRef:
-          name: secret-name
-          namespace: namespace-name
-        keys:
-        - name: first-key
-  ```
+* `images`: The images block defines the patterns of images that should be
+  subject to the ClusterImagePolicy. If multiple policies match a particular
+  image, _ALL_ of those policies must be satisfied for the image to be admitted.
+  If there is no host in the `glob` field, `index.docker.io` is used for the
+  host.
 
-The `name` for the `ClusterImagePolicy` resource must be `image-policy`.
+* `authorities`: The authorities block defines the rules for discovering and
+  validating signatures. Discovery is done through the `sources` field, and can
+  be specified on any entry. Signatures are cryptographically verified using one
+  of the `key` or `keyless` fields.
 
-Add any namespaces that run container images that are not signed in the
-`spec.verification.exclude.resources.namespaces` section, such as the
-`kube-system` namespace.
+When a policy is selected to be evaluated against the matched image, the
+authorities will be used to validate signatures. If at least one authority is
+satisfied and a signature is validated, the policy is validated.
 
-If no `ClusterImagePolicy` resource is created, all images are admitted into
-the cluster with the following warning:
+### <a id="cip-images"></a> `images`
 
-  ```console
-  Warning: clusterimagepolicies.signing.apps.tanzu.vmware.com "image-policy" not found. Image policy enforcement was not applied.
-  ```
+The ClusterImagePolicy specifies spec.images which specifies a list of glob
+matching patterns. These matching patterns will be matched against the image
+digest of PodSpec resources attempting to be deployed.
 
-The patterns are evaluated using the any of operator to admit container
-images. For each pod, the Image Policy Webhook iterates over the list of
-containers and init containers. The pod is verified when there is at least
-one key specified in `spec.verification.images[].keys[]` for each container image
-that matches `spec.verification.images[].namePattern`.
+Glob uses golang filepath semantics for matching the images against. To make it
```

Comment:
This has changed. it doesnt follow filepath semantics anymore. It is more "glob" like
i.e. 
- `**` is handled like `.*` in regex
- `*` is handled like `[^/]*` in regex
- `.` is handled like `\.` in regex

Filepath globbing did not support multiple `*` for example.